src/test/ui/rust-2018/uniform-paths/import-builtin-macros-stability.rs

```diff
@@ -0,0 +1,10 @@
+// FIXME: Individual imports of built-in macros are not stability checked right now,
+// so the whole feature is gated instead.
```

Comment:
I don't understand what this comment means. Can you illustrate the difference?

Reply:
`use concat_idents as unstable;` below doesn't produce a "`concat_idents` is unstable" error.

Reply:
Oh I see; I read `.stderr` sloppily... thanks!

Reply:
Ahaha, `concat_idents` and friends are actually stable *themselves*, they just report feature errors as a part of their expansion.
I'll fix this, but https://github.com/rust-lang/rust/pull/61898 is a pre-requisite.